/**:
  ros__parameters:
    jpeg_quality: 30  # from 0 to 100 (the higher is the better).
